test/CMakeLists.txt

```diff
@@ -118,6 +129,11 @@ list(APPEND test_kinds 4 8 d)
 
 set(fortran_8_defs INTSIZE_8)
 
+# Add tests which don't require output log processing.
+foreach(kind ${kinds})
+  create_test(intest1 ${kind})
+endforeach()
+
 # IN tests
```

Comment:
This needs to be `foreach(kind, ${test_kinds})`, so that each test code gets built and tested as _4, _8, and _d (i.e. 3 CI tests per test code, just like for all existing IN and OUT tests).  That way, we confirm that each tests code works correctly under each of the 3 possible existing build scenarios.  It's only the library itself that's now consolidated down to one build, which is why we need to hardcode `bufr_4` above.

Reply:
OK, let me take a look...